packages/manager/core/generator/app/conditional-translations/onboarding/Messages_fr_FR.json.hbs

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "{{dashCase appName}}",
+  "description": "Découvrez des services de stockage managés qui s’appuient sur le système de fichiers OpenZFS. Bénéficiez en quelques clics d’espaces de stockage centralisés pour entreposer ou sauvegarder vos données et fichiers.",
+  "orderButtonLabel": "Commander un {{dashCase appName}}",
+  "orderButtonLink": "/#/dedicated/{{appName}}/order",
```

Comment:
**question:** Why are links added in trans files? We will have to duplicate it in all language files.

Reply:
It will be delete in an other ticket, we have a story to manage the guide inside react project in the backlog

Reply:
we have a story in the next sprint concerning the guide, it will be delete